micmac_li3ds/api.py

```diff
@@ -46,6 +49,21 @@ def get_object_by_name(self, typ, obj_name):
               resp.status_code)
         raise RuntimeError(err)
 
+    def get_object_by_dict(self, typ, dict):
```

Comment:
The convention in Python is to use a trailing underscore to avoid conflicts with Python keywords. So I'd suggest that you use `dict_` rather than `dict` here to avoid problems.